applications/minotari_node/src/stratum/job.rs

```diff
@@ -0,0 +1,29 @@
+use chrono::NaiveDateTime;
+use serde::{Deserialize, Serialize};
+use tari_common_types::tari_address::TariAddress;
+use tari_transaction_components::tari_proof_of_work::PowAlgorithm;
+
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct Job {
+    pub id: String,
+    pub job_id: String,
+    pub created_at: NaiveDateTime,
+    pub algo: String,
+    pub pow_algo: PowAlgorithm,
+    pub target: String,
+    pub blob: Vec<u8>,
+    pub height: u64,
+    pub chain_target: String,
+    pub miner_address: TariAddress,
+    pub original_mining_hash: Vec<u8>,
```

Comment:
Why not `HashOutput` here?
```suggestion
    pub original_mining_hash: Vec<u8>,
```